engine/src/test/java/io/zeebe/engine/processing/workflowinstance/CancelWorkflowInstanceConcurrentlyTest.java

```diff
@@ -141,10 +141,16 @@ public void init() {
             .withElementType(BpmnElementType.SERVICE_TASK)
             .getFirst();
 
+    // wait for both jobs to appear
+    assertThat(
+            RecordingExporter.jobRecords(JobIntent.CREATED)
+                .withWorkflowInstanceKey(workflowInstanceKey)
+                .limit(2))
+        .hasSize(2);
```

Comment:
This assertion fails in the CI. Please have a look.

Reply:
Our assumption was wrong. Should be fixed now